File: E4/E04-01/app/src/main/java/com/example/e04_01/data/Employee.java
package com.example.e04_01.data;

public class Employee {
    public String username;
    public String email;

    public Employee() {
        // Default constructor required for calls to DataSnapshot.getValue(User.class)
    }

    public Employee(String username, String email) {
        this.username = username;
        this.email = email;
    }

}

File: 2021-SOPM/E4/E04-01/app/src/main/java/com/example/e04_01/data/Employee.kt
package com.example.e04_01.data

class Employee {
    var username: String? = null
    var email: String? = null

    constructor() {
        // Default constructor required for calls to DataSnapshot.getValue(User.class)
    }

    constructor(username: String?, email: String?) {
        this.username = username
        this.email = email
    }
}